### PR TITLE
✨Ice theme

### PR DESCRIPTION
**Pull request by donut28**
Nice new Ice theme. Accessibility might be an issue, but I've made the text bold so it's all easier to see.

### CODE REVIEW

**Review by no1crate — APPROVED** [review]
This, this is art, this is a level of beautifulness that we can't understand. 10/10

**Comment by donut28** [comment]
I'm experimenting with rounding but other than that, it should be ready in 30 mins - 1 hr

**Comment by donut28** [comment]
It's ready! 🎉🎉

File: themes/dark/terminal.css
Comment (kurmachu) [inline themes/dark/terminal.css:1]: Accessibility optimized themes are reviewed to a higher standard than the default accessibility level. They should be free of _all_ accessibility issues (up to AAA) on all states (hover, active, etc)
image
Further review has been paused until this instance is resolved, or until accessibility is set back to standard. 
```suggestion
/* [THEME META] {"ver":1, "author":"donut28", "description":"Feel like a hacker with this customised dark theme."} */
```

File: themes/light/ice.css
Comment (kurmachu) [inline themes/light/ice.css:1]: Making a theme bold does not make it accessible (According to WebAIM). While gradients do help visually, they are not pronounced enough here (and there are some almost-solid colors failing AA). Please opt-out, or update theme.
```suggestion
/* [THEME META] {"ver":1, "author":"donut28", "description":"Blue but with Discord light mode mixed in. Will make you blind.", "accessibility","ignore"} */
```

**Review by kurmachu — CHANGES_REQUESTED** [review]

**Comment by donut28** [comment]
Fixed!

**Comment by donut28** [comment]
Did an oopsie with Git and it merged it so it doesnt have accessibility.

Oops...

**Comment by kurmachu** [comment]
Alright, waiting for request to review.

**Comment by donut28** [comment]
> Alright, waiting for request to review.

Should be okay now. Git is very confusing.

**Review by kurmachu — CHANGES_REQUESTED** [review]
As stated, accessible themes "should be free of all accessibility issues (up to AAA) on all states (**hover**, active, etc)"
image
Further review has been paused until this is resolved.
To get this resolved faster, you should check anything you're unsure on before requesting a review.
You can find the online checker and documentation on the valid ratios at: https://webaim.org/resources/contrastchecker/
_Thank you for your commitment to accessibility!_

ℹ️ Standard themes _do not_ need to remain accessible during hovers.
✅ **Ice** is approved and can be merged once this is resolved.

**Comment by donut28** [comment]
All my checks pass. Should be complete now.

**Comment by donut28** [comment]
This turned from a nice new light theme to the logistics of accessibility on the internet lol
Oh well, its good to have accessibility features in place

**Review by kurmachu — CHANGES_REQUESTED** [review]
> Accessibility optimized themes are reviewed to a higher standard than the default accessibility level. They should be free of all accessibility issues (up to **AAA**) on all states (hover, active, etc)

image

**Comment by no1crate** [comment]
@donut28 I'm just saying. It's easier ignoring that! Haha

**Comment by donut28** [comment]
I'm dedicated to this being the first accessibility theme. This should cover it.

File: themes/dark/terminal.css
Comment (kurmachu) [inline themes/dark/terminal.css:89]: What uh does this actually do? Nothing is wrong here just confused.

**Review by kurmachu — CHANGES_REQUESTED** [review]
image

**Comment by donut28** [comment]
🙏 code... please pass the checks... please

**Comment by donut28** [comment]
☕ Morning,
I have checked every part of the document and have tested it. This should pass.

**Review by kurmachu — APPROVED** [review]
✅ All tests passed, congratulations!
_Thank you for your commitment to accessibility._

**Pull request merged**
